desk-controller/main.py

```
@@ -1,120 +1,67 @@
     try:
-        mainApp.mount('/height', height_server)
-        mainApp.mount("/time", time_server)
-        mainApp.mount("/setting", setting_server)
-
-        # executes enable_cors on all routes (https://stackoverflow.com/questions/17262170/bottle-py-enabling-cors-for-jquery-ajax-requests)
-        mainApp.install(EnableCors())
-        mainApp.run(host="0.0.0.0", port=8080)
+        logger.info("starting desk_controller ...")
+        desk_state.set_height(http_controller.get_current_height())
+        get_current_height_loop()
     except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)